web/package.json

```diff
@@ -10,28 +10,28 @@
     "build": "vite build",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "fmt": "npx prettier --write *.html src/**",
+    "fmt": "npx prettier --write *.js *.ts *.json *.html src/**",
     "list-tests": "./public/list_tests.sh"
   },
   "devDependencies": {
     "@ianvs/prettier-plugin-sort-imports": "^4.0.2",
     "@sveltejs/vite-plugin-svelte": "^2.4.2",
-    "@tsconfig/svelte": "^4.0.1",
+    "@tsconfig/svelte": "^5.0.2",
     "@types/mapbox__mapbox-gl-draw": "^1.4.0",
-    "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.1",
+    "prettier": "^3.2.5",
+    "prettier-plugin-svelte": "^3.2.2",
     "svelte": "^4.0.0",
     "svelte-check": "^3.4.4",
     "tslib": "^2.6.0",
     "typescript": "^5.1.3",
     "vite": "^4.3.9",
-    "vite-plugin-wasm-pack": "0.1.11"
+    "vite-plugin-wasm-pack": "^0.1.12"
```

Comment:
Huh, weird. I had kept this at 0.1.11 because I had some problem with 0.1.12 before, but trying this out locally, it's all working fine. Cool.

Reply:
That must've updated automatically by `npm update`. I didn't update that one manually.